pkg/util/errors.go

```diff
@@ -282,6 +285,24 @@ func HandleInstanceSharingError(err error, entityName string) error {
 			Description: fmt.Sprintf("Failed to create the reference. The instance %s, for which you want to create the reference, does not match the service offering.", entityName),
```

Comment:
"Failed to create the reference. The Instance %s, for which you want to create the reference, belongs to a different service."

Reply:
do you mean "The instance"? 
you wrote "The Instance"